packages/components/src/spinner/README.md

```diff
@@ -1,5 +1,16 @@
 # Spinner
 
+Spinners notify users that their action is being processed.
+
+![Spinner component](https://wordpress.org/gutenberg/files/2019/07/spinner.png)
+
+## Best practices
+
+The spinner component should:
+
+- Signal to users that the processing of their request is underway and will soon complete.
+- Not appear when pages or data are loading.
```

Comment:
Can you clarify this `Not appear when pages or data are loading.`? Isn't this the whole purpose of the spinner component (data being loaded)?

Reply:
Yep this is a good question. The thinking here is that future forward it's ideal to have different kinds of feedback or progress indicators, each with their own purpose. Since the spinner is the only progress indicator we have I jumped the gun with this guidance. 

Future component descriptions might look something like this:

**Spinner:** An action is being processed with an unspecified amount of wait time. They should be used when progress isn’t detectable, or if it’s not necessary to indicate how long an activity will take.

**Progress bar:** Used to indicate to users that a page is loading or an upload is processing. It shows how much of the task has been completed and how much is left.

In the meantime I can remove `Not appear when pages or data are loading` until we hopefully add a Progress bar component.